tests/e2e-cypress/integration/search/search-results.cy.js

```diff
@@ -301,7 +304,7 @@ describe('Test the search bar result handling', () => {
         cy.get(searchbarSelector).paste('test')
         cy.wait(`@search-locations`)
         cy.get('[data-cy="search-result-entry-location"]').should('be.visible')
-        cy.get('[data-cy="map"]').click()
+        cy.get('[data-cy="map"]').click(viewportWidth * 0.5, viewportHeight * 0.75)
```

Comment:
was there an issue while clicking at the center of the map?

Reply:
Yes as the search results go a bit further down than half the screen, so clicking on the center would actually not click on the map.